### PR TITLE
[TBCCT-142] Removed ID columns from all records / tables

### PR DESCRIPTION
**Pull request by onehanddev**
Removed ID columns from all records / tables.

### CODE REVIEW

File: admin/resources/conservation-and-planning-admin/conservation-and-planning-admin.resource.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { ResourceWithOptions } from "adminjs";
 import { ConservationPlanningAndAdmin } from "@shared/entities/cost-inputs/conservation-and-planning-admin.entity.js";
+import { GLOBAL_COMMON_PROPERTIES, COMMON_RESOURCE_LIST_PROPERTIES } from "../common/common.resources.js";
 
 export const ConservationAndPlanningAdminResource: ResourceWithOptions = {
   resource: ConservationPlanningAndAdmin,
   options: {
+    properties: {...COMMON_RESOURCE_LIST_PROPERTIES, ...GLOBAL_COMMON_PROPERTIES},
```
Comment (alexeh) [inline admin/resources/conservation-and-planning-admin/conservation-and-planning-admin.resource.ts:8]: In this case this entity does not have a activity column, but it renders in the UI

**Review by alexeh — CHANGES_REQUESTED** [review]
Thanks @onehanddev!

I just realise there is a very minor bug, where you have added a property in the resource configuration for an entity that does not have said property (I pointed it out)

Which basically makes this happen:
image


Also, there are 3 more entities that we need to remove the ID from:
image

These probably where added after you started working on this, please rebase latest dev to have them

Hit me up through Slack whenever you manage to do this and let's move on to next tasks
Thanks!

File: admin/resources/conservation-and-planning-admin/conservation-and-planning-admin.resource.ts (hunk 1 above)
Comment (onehanddev) [reply admin/resources/conservation-and-planning-admin/conservation-and-planning-admin.resource.ts:8]: Ah, I see, I was trying something out and it remained that way, Removing COMMON_RESOURCE_LIST_PROPERTIES  from the properties. 👍

**Comment by onehanddev** [comment]
@alexeh TBCCT-142 is also fixed in this PR. Can you please that as well. Thanks.

**Pull request merged**